zds/member/commons.py

```diff
@@ -306,6 +306,7 @@ def get_detail(self):
     def apply_sanction(self, profile, ban):
         profile.end_ban_read = None
         profile.can_read = False
+        profile.biography = u"Biographie masquée car le membre est bannie"
```

Comment:
Pas possible sans changement de modèle. Et overkill par rapport au 
besoin réel : on ne banni pas un membre pour le plaisir.


Le 20/06/2016 à 22:03, Amaury Carrade a écrit :
>
> In zds/member/commons.py 
> <https://github.com/zestedesavoir/zds-site/pull/3676#discussion_r67756334>:
>
> > @@ -306,6 +306,7 @@ def get_detail(self):
> >      def apply_sanction(self, profile, ban):
> >          profile.end_ban_read = None
> >          profile.can_read = False
> > +        profile.biography = u"Biographie masquée car le membre est bannie"
>
> « ...car le membre est /banni/* »
>
> Ne faudrait-il pas conserver le texte mais la masquer au public, 
> plutôt que le perdre comme ici ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3676/files/f340b7ba71820683fc3fae3e281a7821872918e6#r67756334>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABc_xQM2d-CBH0qVmg3EqUrdn1ZYZv91ks5qNvImgaJpZM4I6Dmw>.
>